db/migrate/20170309125642_add_alert_ems_ref.rb

```diff
@@ -0,0 +1,6 @@
+class AddAlertEmsRef < ActiveRecord::Migration[5.0]
+  def change
+    add_column :miq_alert_statuses, :ems_ref, :string
+    add_index :miq_alert_statuses, :ems_ref
```

Comment:
@kbrock I think a good solution might be to have an index on event_ems_ref and remove the resource part from the query. WDYT?

https://github.com/moolitayer/manageiq/blob/f423f61dbc0485eee72f2bb153ce1303e70a96dd/app/models/miq_alert.rb#L217